src/main.cpp

```diff
@@ -2,7 +2,7 @@
 
 int main(int argc, char **argv)
 {
-    auto test = new std::string;
+    auto test = new std::string;  // :)
     std::cout << "Hello xD\n";
```

Comment:
warning: Potential leak of memory pointed to by 'test' [clang-analyzer-cplusplus.NewDeleteLeaks]
```cpp
    std::cout << "Hello xD\n";
    ^
```
**/home/pajlada/git/mini-cpp-project/src/main.cpp:4:** Memory is allocated
```cpp
    auto test = new std::string;  // :)
                ^
```
**/home/pajlada/git/mini-cpp-project/src/main.cpp:5:** Potential leak of memory pointed to by 'test'
```cpp
    std::cout << "Hello xD\n";
    ^
```